tests/lib/snaps/apply-config-hook-set/meta/hooks/apply-config

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo "Setting a configuration value"
+if ! snapctl set baz=qux; then
```

Comment:
Haha, is it sad that such an idea (using config) never occurred to me? Yeah let me give that a shot.